.tekton/pr-endpoint-run-aro.yaml

```diff
@@ -7,7 +7,7 @@
 apiVersion: tekton.dev/v1
 kind: PipelineRun
 metadata:
-  generateName: endpoint-tests-
+  generateName: endpoint-tests-aro-
```

Comment:
@mjiao I think this variable should be modified a bit, ideally variable names should not end with "-", this could lead to issues related to calling commands in the CLI or elsewhere.